server/container_create_linux.go

```diff
@@ -1079,12 +1079,12 @@ func getOCIMappings(m []*types.IDMapping) []rspec.LinuxIDMapping {
 		return nil
 	}
 	ids := make([]rspec.LinuxIDMapping, 0, len(m))
-	for i, m := range m {
-		ids[i] = rspec.LinuxIDMapping{
+	for _, m := range m {
+		ids = append(ids, rspec.LinuxIDMapping{
```

Comment:
Would it be easier to do this change instead?
```diff
-	ids := make([]rspec.LinuxIDMapping, 0, len(m))
+	ids := make([]rspec.LinuxIDMapping, len(m))
```

Reply:
As far as I understand the intent of the original implementer of this function: `make([]rspec.LinuxIDMapping, len(m))` would already initialize the elements of the underlying array and be therefore slightly less efficient than `make([]rspec.LinuxIDMapping, 0, len(m))`.  

The resulting `ids` should be exactly the same.